packages/react-components/react-avatar/src/components/AvatarGroupPopover/AvatarGroupPopover.types.ts

```diff
@@ -55,7 +55,7 @@ export type AvatarGroupPopoverProps = Omit<ComponentProps<Partial<AvatarGroupPop
  * State used in rendering AvatarGroupPopover
  */
 export type AvatarGroupPopoverState = ComponentState<AvatarGroupPopoverSlots> &
-  Required<Pick<AvatarGroupPopoverProps, 'indicator'>> & {
+  Required<Pick<AvatarGroupPopoverProps, 'indicator' | 'count'>> & {
```

Comment:
_nit:_ Order alphabetically

```suggestion
  Required<Pick<AvatarGroupPopoverProps, 'count' | 'indicator'>> & {
```